README.md

```diff
@@ -22,6 +22,13 @@ through setting up and using TensorBoard. There's an associated
 
 [tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/martinwicke/tf-dev-summit-tensorboard-tutorial
 
+You may also be interested in the hosted TensorBoard solution at
```

Comment:
I'd suggest mentioning the value of this product. I.e. "You can use [TensorBoard.dev](https://tensorboard.dev) to easily host, track, and share your ML experiments for free. For example, ..."

Reply:
Added text regarding why you might be interested in tensorboard.dev